docs/esp8266/general.rst

```diff
@@ -151,25 +151,26 @@ SSL/TLS limitations
 ~~~~~~~~~~~~~~~~~~~
 
 ESP8266 uses `axTLS <http://axtls.sourceforge.net/>`_ library, which is one
-of the smallest TLS libraries with the compatible licensing. However, it
+of the smallest TLS libraries with compatible licensing. However, it
 also has some known issues/limitations:
 
 1. No support for Diffie-Hellman (DH) key exchange and Elliptic-curve
-   cryptography (ECC). This means it can't work with sites which force
-   the use of these features (it works ok with classic RSA certifactes).
+   cryptography (ECC). This means it can't work with sites which require
+   the use of these features (it works ok with the typical sites that use
+   RSA certificates).
 2. Half-duplex communication nature. axTLS uses a single buffer for both
    sending and receiving, which leads to considerable memory saving and
    works well with protocols like HTTP. But there may be problems with
    protocols which don't follow classic request-response model.
 
-Besides axTLS own limitations, the configuration used for MicroPython is
+Besides axTLS's own limitations, the configuration used for MicroPython is
 highly optimized for code size, which leads to additional limitations
 (these may be lifted in the future):
 
 3. Optimized RSA algorithms are not enabled, which may lead to slow
    SSL handshakes.
-4. Stored sessions are not supported (may allow faster repeated connections
-   to the same site in some circumstances).
+4. Session Reuse is not enabled, which means every connection must undergo
```

Comment:
Google "SSL session reuse": 362,000, "SSL stored sessions": 463,000. But ok, the term appears to be the 1st one.